src/less/components/form.less

```
@@ -211,25 +185,23 @@ legend {
 }
 
 .form-control {
-    height: @input-form-control-height;
-    padding: 0 0.75rem;
-    background-color: @input-bg;
-    background-clip: padding-box;
-    border: solid @input-border-color;
-    border-width: @input-border-width;
-    border-bottom-color: @medium-brown;
-    border-radius: 2px 2px 0 0;
-    box-shadow: none;
-
-    .form-control-focus(@brand-secondary);
-    .placeholder(@brand-secondary-light-2);
-
-    &::placeholder {
-        color: @brand-secondary-light-2; /* Override Firefox's unusual default opacity; see https://github.com/twbs/bootstrap/pull/11526. */
-        font-weight: @font-weight-normal;
-        letter-spacing: -0.1px;
+    height: 56px;
+    padding: 12px 16px;
+    background-color: var(--white);
+    border: 1px solid var(--input-border-color);
+    border-radius: 8px;
```

use the css variable?
```suggestion
            border-radius: var(--border-radius, 8px);
```